ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env

```diff
@@ -0,0 +1,52 @@
+HOST={{sl_usermanagement_host|default("")}}
+APPLICATION_PORT={{sl_usermanagement_application_port|default('4101')}}
+LOG={{sl_usermanagement_log|default('debug')}}
```

Comment:
Please add one line comment for all the variables
for example

```

# Connection string to the mysql db
# format: username@password:ip:port
secret_variable_connection_db: {{ db_connection_string | default ("") }}
```

Reply:
yes rajesh changes had been done kindly review it

Reply:
Hi rajesh, sl usermanagement devolper ask me remove, i am removing this one, and i am making pull request for sl sunbird